leaderboard/database/redis.go

```diff
@@ -0,0 +1,209 @@
+package database
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/topfreegames/podium/leaderboard/database/redis"
+)
+
+// Redis is a type that implements database interface with redis client
+type Redis struct {
+	redis.Redis
+}
+
+var _ Database = &Redis{}
```

Comment:
Why are you creating an anonymous variable here?

Reply:
This is the way to assert at compile time that a type implements a certain interface. In this case, I am asserting that Redis implements Database interface

Reply:
Shouldn't this be done in a test file?

Reply:
Very interesting btw

Reply:
Hm, how other private types as `clusterClient` and `standaloneClient` inside `redis` package should do this same kind of assertion, and test is on "another" package . I prefer to keep all type assertions inside the same file.

Reply:
https://github.com/topfreegames/podium/pull/45/files#diff-ba0a1ca105e821459b703bf496c548b2401d1ce3e0ca43f60cd0e449130d01fdR15